docs/sources/reference/api/docker_remote_api.md

```diff
@@ -52,6 +52,21 @@ You can still call an old version of the API using
 You can now supply a `stream` bool to get only one set of stats and
 disconnect
 
+`GET /images/json`
+
+**New!**
+The `Created` field will be responsed as ISO 8601 String instead of UNIX epoch
```

Comment:
RFC 3339 is listed as a profile of ISO 8601, do you think it's better to use RFC3339?